docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,102 @@
-// TODO #30642 A doc explaining day to day dev work flow.
+## Making changes
 
-Assignee - Mason
-Reviewer - Sarah
+### Creating a branch
+
+Create a branch from your forked repo for your code changes:
+
+\```
+git checkout master
+git pull upstream master
+git checkout -b my-branch-name
+\```
+
+We strongly recommend using the CLI as your primary interface with git. GUIs are useful for viewing diffs, creating branches and making quick commits. However they often do not correctly merge and sync working branches with master. Mistakes in this step are time consuming to fix.
+
+That said, some team members use [SourceTree](https://www.sourcetreeapp.com/), [GitHub app](https://desktop.github.com/) or the git integrations available in VS Code.
+
+We don't have an official branch naming policy. However some developers use the following to make their branches easier to find: `user/[alias]/your-fancy-branch-name`, ie `user/johndoe/fix-that-bug`.
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time.
+
+```
+cd fluentui
+yarn
+```
+
+Run `yarn start` and select your start up project.
```

Comment:
note: once this lands https://github.com/microsoft/fluentui/issues/30267 this command won't do anything/will be probably completely removed.

Reply:
Noted :-)